commit f49200c36738a315eb094a3ed8260f00d1bafee0
Author: nohee ko <noheek@broadcom.com>
Date:   Sun Oct 17 10:33:19 2010 -0700

    staging: brcm80211: brcmfmac: remove unnecessary cflags, WL_ISCAN_DISABLED & WL_ROAM_DISABLED
    
    Remove unnecessary cflags, WL_ISCAN_DISABLED
    and WL_ROAM_DISABLED. Iscan is ON and Roam by FW
    is OFF by default.
    
    Signed-off-by: Nohee Ko <noheek@broadcom.com>
    Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

--- a/drivers/staging/brcm80211/brcmfmac/wl_cfg80211.c
+++ b/drivers/staging/brcm80211/brcmfmac/wl_cfg80211.c
@@ -3108,18 +3108,10 @@ static s32 wl_init_priv(struct wl_priv *
 
 	wl->scan_request = NULL;
 	wl->pwr_save = !!(wiphy->flags & WIPHY_FLAG_PS_ON_BY_DEFAULT);
-#ifndef WL_ISCAN_DISABLED
 	wl->iscan_on = true;	/* iscan on & off switch.
 				 we enable iscan per default */
-#else
-	wl->iscan_on = false;
-#endif				/* WL_ISCAN_DISABLED */
-#ifndef WL_ROAM_DISABLED
-	wl->roam_on = true;	/* roam on & off switch.
+	wl->roam_on = false;	/* roam on & off switch.
 				 we enable roam per default */
-#else
-	wl->roam_on = false;
-#endif				/* WL_ROAM_DISABLED */
 
 	wl->iscan_kickstart = false;
 	wl->active_scan = true;	/* we do active scan for
